assets/css/styles.css

```diff
@@ -0,0 +1,36 @@
+@import url('https://fonts.googleapis.com/css?family=Rubik:300,400&display=swap');
+
+body{ 
+    width: 100%; 
+    height: 100%; 
+    margin: 0px; 
+    padding: 0px; 
+    background: #E5E5E5;
+}
+
+*{
+    font-family: 'Rubik', sans-serif !important;
+    font-weight: 300 !important;
+}
+
+.form-container{
+    padding: 4em 3em;
+    border-radius: 0.5em;
+    margin: auto;
+    margin-top: 2.6em;
+    background-color: #ffffff;
+}
+
+.logo{
+    margin: auto;
+    height: 3.5em;
+    width: 6em;
+}
+
+.submit-btn{
+    background: #990D0D;
+    color: #ffffff;
+    text-transform: uppercase;
+    font-size: small;
+    padding: 1em 2em;
+}
```

Comment:
Nit: Ensure you add an empty newline to the end of each of your files. Please see this for more information - https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline